docs/content/deployment/guides/kubernetes/deploying-with-helm.mdx

```diff
@@ -135,7 +135,7 @@ In the following tutorial, we install the most recent version of the Dagster Hel
 </TabItem>
 <TabItem name="Code location server">
 
-A [code location server](/concepts/code-locations) runs a gRPC server and responds to Dagit's requests for information, such as:
+A [code server](/concepts/code-locations) runs a server that responds to Dagit's requests for information, such as:
 
 - List all of the jobs in each code location, or
 - What is the dependency structure of job X?
```

Comment:
I can't comment on the line because it wasn't changed, but line 152 in this page mentions gRPC:

> The Dagit webserver communicates with user code deployments via gRPC to fetch information needed to populate the Dagit UI.

Does it need to be?

Reply:
:+1: Works for me; just wanted to check! I'll stamp this guy